src/app/clusters/ota-requestor/OTARequestor.cpp

```diff
@@ -36,6 +36,7 @@ using namespace app::Clusters::OtaSoftwareUpdateProvider;
 using namespace app::Clusters::OtaSoftwareUpdateProvider::Commands;
 using namespace app::Clusters::OtaSoftwareUpdateRequestor;
 using namespace app::Clusters::OtaSoftwareUpdateRequestor::Commands;
+using namespace DeviceLayer;
```

Comment:
Suggest adding `DeviceLayer::` namespace directly to `ConfigurationManager` access to make clear where it comes from